Dockerfile

```diff
@@ -109,6 +109,7 @@ RUN --mount=type=cache,target=/root/.cache/pip \
             export SCCACHE_BUCKET=vllm-build-sccache; \
            fi \
         && export SCCACHE_REGION=us-west-2 \
+        && export SCCACHE_S3_NO_CREDENTIALS=1 \
```

Comment:
would this limit the use of sccache bucket to readonly? I think builds still need to write to the bucket

Reply:
nice catch. sorry my old change go carried over (btw this is very useful for local build)
```suggestion
```

Reply:
oh yes it'd be helpful to access the cache bucket locally. Can we mention it in our documentation?